mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/processing/impl/TypeBindingProcessor.java

```diff
@@ -28,8 +29,16 @@
 	@Override
 	public void process(TypeMappingStep mapping, TypeBinding annotation,
 			TypeMappingAnnotationProcessorContext context) {
-		TypeBinder binder = createBinder( annotation.binder(), context );
-		mapping.binder( binder );
+		TypeBinderRef typeBinder = annotation.binder();
+		TypeBinder binder = createBinder( typeBinder, context );
+
+		if ( typeBinder.params() != null ) {
+			Map<String, Object> params = ParameterAnnotationsReader.toMap( typeBinder.params() );
```

Comment:
What's the point of exposing a method in the `context` if you don't use it? :)

```suggestion
			Map<String, Object> params = context.toMap( typeBinder.params() );
```